daemon/daemon.go

```diff
@@ -668,6 +687,7 @@ func (daemon *Daemon) shutdownContainer(c *container.Container) error {
 	}
 
 	c.WaitStop(-1 * time.Second)
+	daemonActions.WithValues("shutdown_container").UpdateSince(start)
```

Comment:
If you want to track how many containers are on each machine, add a gauge to keep track. Doing it off counters can be slightly inaccurate.